wormhole-foundation/wormhole
# sui: add `MigrateComplete` event; clean up

## Objective

Added an event when `migrate` is successfully executed on Wormhole and Token Bridge (called `MigrateComplete`) which will emit information about the package whose version is now valid. Added checks that the event was emitted after `migrate` was called.

Included in this PR is some refactoring by moving more logic to `wormhole::package_utils`. And some clean up of comments are included, too.

## How to Review PR
- Make sure Move tests run via `make test`
- Review `migrate` tests.
- Make sure modified comments make sense.
